typescript/forms.d.ts

```diff
@@ -1108,6 +1108,8 @@ export type NewForm = {
   }
   /** Disable form data being autosaved */
   disableAutosave?: boolean
+  /** Whether or not the form is archived. Only archived forms can be deleted */
```

Comment:
```suggestion
  /** Whether or not the form is archived. */
```

I think (Dave and Josh agree) the  *Only archived forms can be deleted* AC should only be enforced in the console. If a developer wants to delete a form, I don't think they should have to make two requests to the API.